### PR TITLE
Feature: Style Cleanup

### PR DESCRIPTION
**Pull request by jaypeasee**
## Is this a fix or a feature?
Feature

## What is the change?
This branch styles the entire desktop view of the page according to the original mock.

## Where should the reviewer start?
Review every single component's SCSS file.

## How should this be tested?
Run `npm start` to load the page to make sure the design matches the comp.

## Relevant Tickets
* #37 
* #36 
* #12 
* #11 
* #9 
* #7 
* #6 
* #5 
* #4 
* #3 
* #2 
* #1 

### Screenshot (optional)
Screen Shot 2021-01-12 at 8 56 32 AM
Screen Shot 2021-01-12 at 8 57 29 AM
Screen Shot 2021-01-12 at 8 58 04 AM

@saraho1123
@jaypeasee
@baileydunning

### CODE REVIEW

File: src/App/App.scss [hunk 1]
```diff
@@ -10,6 +9,10 @@
   background-color: $gray;
 }
 
+body {
+  margin: 0;
```
Comment (baileydunning) [inline src/App/App.scss:13]: Really glad we figured out where to get rid of this margin!

File: src/Footer/Footer.scss [hunk 2]
```diff
@@ -2,23 +2,26 @@
...
-  padding: 10px;
-  margin: 0px 60px;
+  padding: 5%;
+  margin: 0 5%;
   text-align: center;
   background-color: $gray;
-  box-shadow: 5px 5px 5px $dark-green;
+  box-shadow: 5px 5px 5px black;
+  &:hover {
+    transform: scale(1.1);
```
Comment (baileydunning) [inline src/Footer/Footer.scss:25]: Love the look with these box shadows!

**Comment by jaypeasee** [comment]
ESLint Cache Merge Conflict is resolved.

File: src/Location/Location.scss [hunk 3]
```diff
@@ -1,4 +1,10 @@
+.map-container {
+  display: flex;
+  justify-content: center;
+}
+
 .map {
   width: 60vw;
   height: 300px;
+  box-shadow: black 5px 5px 5px;
```
Comment (saraho1123) [inline src/Location/Location.scss:9]: I love how you did the box shadows!

File: src/ParkBtn/ParkBtn.scss [hunk 4]
```diff
@@ -2,7 +2,17 @@
 @import '../variables.scss';
 
 .park-button {
+  padding: 5% 0;
+  width: 100%;
```
Comment (baileydunning) [inline src/ParkBtn/ParkBtn.scss:6]: These buttons look so much better when they fill up the MainNav

**Review by baileydunning — APPROVED** [review]

File: src/ParkList/ParkList.scss [hunk 5]
```diff
@@ -11,8 +11,16 @@
 
 .randomize-button {
   cursor: pointer;
+  width: 80%;
+  margin-bottom: 4%;
+  background-color: $gray;
+  box-shadow: 5px 5px 5px black;
+  &:hover {
+    transform: scale(1.05);
+    transition-duration: .4s;
```
Comment (saraho1123) [inline src/ParkList/ParkList.scss:20]: I love this transition! Thanks Bailey!

**Review by saraho1123 — APPROVED** [review]
It looks great! Thanks for all the new css tricks you both taught me!

**Pull request merged**